### PR TITLE
Connectivity Test clean up

### PR DESCRIPTION
**Pull request by philipktlin**
Clean up, fix warnings and add more log for connectivity test investigation.

### CODE REVIEW

File: edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs [hunk 1]
```diff
@@ -23,8 +23,6 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IController, EdgeletTestImplementation>();
         }
 
-        // TODO: Remove warning disable for Obsolete when project is moved to dotnetcore 3.0
-#pragma warning disable 612, 618
```
Comment (philipktlin) [inline edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs]: Removed and built locally, and didn't see any warnings; therefore took it out.

File: edge-modules/TestAnalyzer/ReportingCache.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@ namespace TestAnalyzer
     using System.IO;
     using System.Linq;
     using System.Threading.Tasks;
-    using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
```
Comment (philipktlin) [inline edge-modules/TestAnalyzer/ReportingCache.cs]: Just run format document for this file.

File: edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs (hunk 1 above)
Comment (ancaantochi) [reply edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs]: Does it build with netcore 2.1 and netcore 3.0?

Comment (philipktlin) [reply edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs]: reverted

**Pull request merged**